CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ if (CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
 endif ()
 
 # Create the library for ablate
-add_library(ablateLibrary SHARED)
```

Comment:
Would you mind moving these into their corresponding cmake files?

Reply:
The main CMake file was messed up somehow. I've updated that. Thanks for pointing it out.